### ISSUE TITLE
Aria messages cannot be localized and one of the messages contains wrong statement

### ISSUE THREAD

**Issue by dmarcautan** [JedWatson/react-select#3352]
The messages returned by 'instructionsAriaMessage' from accessibility/index are in plain English and there's no option to localize them. Also, when a value in multi-option setting is focused the messages returned states that item can be removed by pressing Enter that is not true.

**Comment by Kashkovsky** [comment]
#3358 Here's the PR for this issue.
